content/about/introduction.md

```diff
@@ -16,7 +16,7 @@ Select a GA release version from the dropdown on the top left:
 
 ![Versions](/images/release-notes/releasenotes-version-dropdown.png)
 
->**Info:** The release notes reach back to GA release 10.4.0. For release notes on older GA releases, please [contact support](https://cumulocity.com/guides/about-doc/contacting-support/).
+>**Info:** The release notes reach back to GA release 10.4.0. For release notes on older GA releases, please [contact support](https://cumulocity.com/guides/welcome/contacting-support/).
```

Comment:
This should point to the contacting-support page in the release notes.

Reply:
Fixed